src/pages/admin/AdminCustomerInstitutionPage.tsx

```diff
@@ -1,10 +1,143 @@
 import React, { FC } from 'react';
 import Card from '../../components/Card';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import { Field, Form, Formik } from 'formik';
+import * as Yup from 'yup';
+import { Button, TextField } from '@material-ui/core';
+import { CustomerInstitutionFieldNames, emptyCustomerInstitution } from '../../types/customerInstitution.types';
+import Heading from '../../components/Heading';
 
+const StyledFieldWrapper = styled.div`
+  margin: 1rem;
+  flex: 1 0 40%;
+`;
+
+const StyledButtonContainer = styled.div`
+  margin-top: 2rem;
+  margin-right: 1rem;
+  display: flex;
+  justify-content: flex-end;
+`;
 const AdminCustomerInstitutionPage: FC = () => {
+  const { t } = useTranslation('admin');
+
+  const onClickSave = () => {};
+
   return (
     <Card>
-      <form>instituition-form</form>
+      <Heading>{t('add_institution')}</Heading>
+      <Formik
+        initialValues={emptyCustomerInstitution}
+        validationSchema={Yup.object({
+          name: Yup.string().required('Required'),
+        })}
+        onSubmit={(values, { setSubmitting }) => {
+          setTimeout(() => {
+            alert(JSON.stringify(values, null, 2));
+            setSubmitting(false);
+          }, 400);
+        }}>
+        <Form>
+          <StyledFieldWrapper>
+            <Field
+              aria-label={CustomerInstitutionFieldNames.NAME}
+              name={CustomerInstitutionFieldNames.NAME}
+              label={t('organizationRegisterName')}
+              component={TextField}
```

Comment:
what, virker dette? sjekka for oppdatering av verdi, fjerning av verdi og validering? mener vi måtte bruke `formik-material-ui`-komponenter for å få det til å funke, men vi har gått mer over til vanilla formik i det siste (feks: https://github.com/BIBSYSDEV/NVA-Frontend/blob/develop/src/pages/publication/references_tab/BookReferenceForm.tsx)

Reply:
men hvis dette virker like bra er det selvsagt en mye finere måte å gjøre det på!

Reply:
skjønte ikke helt hva du mente. men du hadde rett i at det ikke funket. men ved å bytte import på textfield til formik-material-ui gikk det som smurt. Prøvde på andre måten(1.) også, men synes det var mindre oversiktelig. dessuten funket det ikke 100%. Feltene ble ikke disabled når alert-boksen vistes f.ex.
(1.) 
<Field name={CustomerInstitutionFieldNames.DISPLAY_NAME}>
              {({ field }: FieldProps) => (
                <MuiTextField .......

Reply:
vi har vært litt frem og tilbake der... har kommet beta for v2 av formik-material-ui også nå, så vi kunne vurdert det også. men kan la det være for nå :)